cmd/internal/loginprovider/tryinorderloginprovider.go

```diff
@@ -26,8 +26,14 @@ func NewTryInOrderLoginProvider(logger loggo.Logger, providers ...api.LoginProvi
 }
 
 type tryInOrderLoginProviders struct {
-	providers []api.LoginProvider
-	logger    loggo.Logger
+	providers  []api.LoginProvider
+	logger     loggo.Logger
+	loginToken string
+}
+
+// Token implements the LoginProvider.Token method.
```

Comment:
As above. Some more comments explaining the specifics of this implementation v satisfying an interface.